web/src/main/java/com/softserve/osbb/dto/adapter/OsbbRegistartionToOsbbAdapter.java

```diff
@@ -17,16 +23,18 @@
     @Override
     public Osbb parse(OsbbRegistrationDTO osbbRegistrationDTO) {
         Osbb registeredOsbb = new Osbb();
+        
         if (osbbRegistrationDTO == null) {
-
             if (dto == null) {
                 throw new IllegalArgumentException();
             }
+            
             osbbRegistrationDTO = (OsbbRegistrationDTO) dto;
             _parse(osbbRegistrationDTO, registeredOsbb);
-            return registeredOsbb;
+        } else {
+            _parse(osbbRegistrationDTO, registeredOsbb);
```

Comment:
this should be wrong .... do you have code convention ? 
usually method names are not starting from underscore symbol .... class fields - yes.